src/coreComponents/physicsSolvers/multiphysics/SinglePhasePoromechanicsKernel.hpp

```diff
@@ -214,108 +214,138 @@ class SinglePhase :
-    real64 dNdX[numNodesPerElem][3];
-    FE_TYPE::calcN( q, N );
-    real64 const detJxW = m_finiteElementSpace.template getGradN< FE_TYPE >( k, q, stack.xLocal, dNdX );
+    // Governing equations (strong form)
+    // ---------------------------------
+    //
+    //   divergence( totalStress ) + bodyForce = 0                       (quasi-static linear momentum balance)
+    //   dFluidMassContent_dTime + divergence( fluidMassFlux ) = source  (fluid phase mass balance)
+    //
+    // with currently the following dependencies on the strainIncrement tensor and pressure
+    //
+    //   totalStress      = totalStress( strainIncrement, pressure)
+    //   bodyForce        = bodyForce( strainIncrement, pressure)
+    //   fluidMassContent = fluidMassContent( strainIncrement, pressure)
+    //   fluidMassFlux    = fludiMassFlux( pressure)
+    //
+    // Note that the fluidMassFlux will dependend on the straiIncrement if a stress-dependent constitutive
+    // model is assumed. A dependency on pressure can also occur in the source term of the mass
+    // balance equation, e.g. if a Peaceman well model is used.
+    //
+    // In this kernel cell-based contributions to Jacobian matrix and residual
+    // vector are computed. The face-based contributions, namely associated with the fluid
+    // mass flux term are computed in a different kernel. The source term in the mass balance
+    // equation is also treated elsewhere.
+    //
+    // Integration in time is performed using a backward Euler scheme (in the mass balance
+    // equation LHS and RHS are multiplied by the timestep).
+    //
+    // Using a weak formulation of the governing equation the following terms are assembled in this kernel
+    //
+    //   Rmom = - \int symmetricGradient( \eta ) : totalStress + \int \eta \cdot bodyForce = 0
+    //   Rmas = \int \chi ( fluidMassContent - fluidMassContentOld) = 0
+    //
+    //   dRmom_dVolStrain = - \int_Omega symmetricGradient( \eta ) : dTotalStress_dVolStrain
+    //                      + \int \eta \cdot dBodyForce_dVolStrain
+    //   dRmom_dPressure  = - \int_Omega symmetricGradient( \eta ) : dTotalStress_dPressure
+    //                      + \int \eta \cdot dBodyForce_dPressure
+    //   dRmas_dVolStrain = \int \chi dFluidMassContent_dVolStrain
+    //   dRmas_dPressure  = \int \chi dFluidMassContent_dPressure
+    //
+    // with \eta and \chi test basis functions for the displacement and pressure field, respectively.
+    // A continuous interpolation is used for the displacement, with \eta continous finite element
+    // basis functions. A piecewise-constant approximation is used for the pressure.
 
-    // Evaluate total stress tensor
     real64 strainIncrement[6] = {0.0};
     real64 totalStress[6];
-    real64 dPorosity_dPressure;
-    real64 dPorosity_dVolStrainIncrement;
+    typename CONSTITUTIVE_TYPE::KernelWrapper::DiscretizationOps stiffness; // Could this be called dTotalStress_dStrainIncrement?
```

Comment:
Strictly speaking this is `dEffectiveStress_dStrainIncrement`.  I think we just used the common term `stiffness` matrix in the constitutive models since it was concise.  Here, you could call it whatever you like.

Reply:
Edit: I see that for the combined models we work in terms of total stress, so disregard the first part of the comment above.  I think the term you proposed is fine here.  I just wouldn't propagate it all the way to the solid mechanics model classes.